libraries/src/Updater/Adapter/ExtensionAdapter.php

```diff
@@ -156,7 +156,7 @@ protected function _endElement($parser, $name)
 						}
 
 						// Do we have an entry for the database?
-						if (\array_key_exists($dbType, $supportedDbs))
+						if (in_array($dbType, array_map('strtolower', array_keys($supportedDbs))))
 						{
 							$minimumVersion = $supportedDbs[$dbType];
```

Comment:
```suggestion
						$dbTypeUcase = strtoupper($dbType);

						if (\array_key_exists($dbTypeUcase, $supportedDbs))
						{
							$minimumVersion = $supportedDbs[$dbTypeUcase];
```
If we do it this way, we can also use the `$dbTypeUcase` below for making the language string key.